content/docs/1_guide/2_read-me/guide.txt

```diff
@@ -0,0 +1,251 @@
+Title: Read me first
+
+----
+
+Description: A quick overview about the different parts of the documentation
+
+----
+
+Intro: Guide, Reference, Cookbook, Search, where to find what and how to best use it
+
+----
+
+
+Text:
+
+Every user comes to a documentation with different expectations, background and prior knowledge. Ideally, Kirby or any other software/product would be self-explanatory, but that's a dream. It is equally a dream to make everyone happy, and consequently some of you will love this documentation and some of you will hate it, as our last survey clearly showed. But that's no excuse, and we will strive to improve it constantly.
```

Comment:
I feel like this paragraph is more for users who already experienced with Kirby (and don't need this guide page). New users might be discouraged by "some of you will hate it".

TBH I feel like we don't need this paragraph at all.